swagger_server/itm/itm_action_handler.py

```diff
@@ -365,6 +365,19 @@ def direct_mobile_characters(self):
         return self.times_dict[ActionTypeEnum.DIRECT_MOBILE_CHARACTERS]
 
 
+    def move_to(self, target_character: Character):
+        """
+        Move to the location of the specified character, toggling whether all characters are seen or not.
+        NOTE: This only works when there are only two locations, which is the stated requirement.
```

Comment:
This comment made me think that maybe characters should have a `location` field. In case there are ever more than two locations in the future, we could modify this for loop to set all of the `unseen` values to `true` for characters without the same location as the `target_character`

Reply:
If we wanted/needed to support more than two locations, then I'm pretty sure you're right-- we'd need a `Character.location` field.  Brian and I are trying to steer clear from >2 locations because we don't really want ADMs to be bouncing all over the scene, having to keep track of where the patients are, spending "time" moving, etc.  But if it becomes a requirement, then we'll implement it and likely add a `location` field, as you suggest.

Reply:
Makes sense